src/phoenix/server/api/types/Span.py

```diff
@@ -1,36 +1,26 @@
 import json
-import math
 from collections import defaultdict
 from datetime import datetime
 from enum import Enum
 from typing import Any, DefaultDict, List, Mapping, Optional, cast
 
 import strawberry
-from pandas import Series
 from strawberry import ID
 from strawberry.types import Info
 
-import phoenix.trace.semantic_conventions as sc
+import phoenix.trace.schemas as s
```

Comment:
More semantic package names are easier to understand IMO (excepting common abbreviations like `pd` and `np` and `tf`).

Reply:
In this case it's just for resolving naming conflicts between the graphql module and our core module. It suffices to keep it short because it serves a very narrow purpose: i.e. as a prefix for disambiguation. The alternative is to rename everything with a `Core` prefix, which looks a bit strange to me (wait, what is a CoreSpan? never seen that before...oh I see).

option 1: Span vs. s.Span (import schema as s)
option 2: Span vs. CoreSpan (from schema import Span as CoreSpan)